R/usethis-package.R

```diff
@@ -10,6 +10,68 @@
 ## usethis namespace: end
 NULL
 
+#' Options consulted by usethis
+#'
+#' @description
+#' User-configurable options consulted by usethis, which provide a mechanism
+#' for setting default behaviours for various functions.
+#'
+#' If the built-in defaults don't suit you, set one or more of these options.
+#' Typically, this is done in the `.Rprofile` startup file, which you can open
+#' for editing with [edit_r_profile()] - this will set the specified options for
+#' all future R sessions. Your code will look something like:
+#'
+#' ```
+#' options(
+#'   usethis.description = list(
+#'     `Authors@R` = 'person("Jane", "Doe", email = "jane@example.com",
+#'       role = c("aut", "cre"), comment = c(ORCID = "YOUR-ORCID-ID"))',
+#'     License = "MIT + file LICENSE",
+#'     Version = "0.0.0.9000"
```

Comment:
Is this `Version` the default? If so, I'd remove it.

If you're looking around for more examples of stuff to set, I think `usethis.overwrite = TRUE` is a good one to publicize.

Reply:
I'll swap out `usethis.overwrite = TRUE` for `usethis.quiet`